src/containers/Chat/index.tsx

```diff
@@ -146,11 +146,27 @@ export class Component extends React.Component<ComponentProps, {}> {
     }
 
     public async navigationButtonPressed({ buttonId }: any) {
-        if (buttonId === 'delete') {
+        const { users } = this.props;
+
+        if (buttonId === 'call') {
+            if (this.props.tree.isPending.get()) {
+                return;
+            }
+
             this.props.tree.isPending.set(true);
+
             try {
-                await removeConversation(this.props.tree.removeConversationResponse, this.props.conversation.uuid);
-                await Navigation.pop(this.props.componentId);
+                try {
+                    await CallService.requestPermissions(false);
+                } catch (err) {
+                    return Navigation.showOverlay({
+                        component: {
+                            name: 'td.Modal',
+                            passProps: { text: err.message },
+                        },
+                    });
+                }
+                await CallService.startOutgoingCall(users[0].username, users[0].displayName);
```

Comment:
@jasiemion `users` it is a list of all contacts. Now you call always to the first user. It is wrong. Try to use a username prop and find the user's display name in the users' list.

Reply:
@ruscoder ok.

Reply:
I didn't found username prop, only userId in conversation object. But problem has been fixed.

Reply:
Additional amendments: variables has been shortened, excess code has been replaced with method makeUsername.